### PR TITLE
Force non-interactive install of git lfs

### PR DESCRIPTION
**Pull request by cd2357**
Update install command to not need user interaction

### CODE REVIEW

File: pricenode/install_pricenode_debian.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ sudo -H -i -u "${ROOT_USER}" DEBIAN_FRONTEND=noninteractive apt-get upgrade -qq
 
 echo "[*] Installing Git LFS"
 sudo -H -i -u "${ROOT_USER}" curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
-sudo -H -i -u "${ROOT_USER}" apt-get install git-lfs
+sudo -H -i -u "${ROOT_USER}" apt-get -y install git-lfs
```
Comment (ripcurlx) [inline pricenode/install_pricenode_debian.sh:39]: @cbeams Do you remember if there was a reason doing it as an interactive install before?

Comment (devinbileck) [reply pricenode/install_pricenode_debian.sh:39]: That was my error. I forgot to include the non-interactive flag.

**Review by devinbileck — APPROVED** [review]
utACK

**Review by ripcurlx — APPROVED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/4382#pullrequestreview-457968351

**Pull request merged**